inspirehep/records/marshmallow/authors/base.py

```diff
@@ -91,3 +91,11 @@ def strip_empty(self, data):
 
 class AuthorsSchemaV1(RecordSchemaJSONV1):
     metadata = fields.Nested(AuthorsMetadataSchemaV1, dump_only=True)
+
+
+class AuthorsMetadataOnlyControlNumberSchemaV1(Schema):
```

Comment:
I'd share this with other types, i.e. literature. It'll be helpful in the future for ``arXiv``,``doi`` etc. It's up to you though